pkg/api/image.go

```diff
@@ -35,7 +35,7 @@ const (
 )
 
 type ImageSpec struct {
-	Size        uint64         `json:"size"`
```

Comment:
I would propose to keep here the `uint64` here since it's the ceph representation of an image and to the handling (`utils.Uint64ToInt64(imageSize)`) at creation time of the image